doc/build-windows.md

```diff
@@ -71,18 +71,33 @@ If you want to build the windows installer with `make deploy` you need [NSIS](ht
 Acquire the source in the usual way:
 
     git clone https://github.com/gridcoin-community/Gridcoin-Research.git
-    git checkout master
     cd Gridcoin-Research
+    git checkout master
 
 ## Building for 64-bit Windows
 
 The first step is to install the mingw-w64 cross-compilation tool chain:
 
     sudo apt install g++-mingw-w64-x86-64
 
-Ubuntu Bionic 18.04:
+Next, set the default `mingw32 g++` compiler option to POSIX<sup>[1](#footnote1)</sup>:
+
+    sudo update-alternatives --config x86_64-w64-mingw32-g++
```

Comment:
This doesn't need to be done on OpenSUSE.

Reply:
(And the documentation states with Ubuntu (or WSL), but this can be successfully done with more than just Ubuntu. Also, WSL 2 is available in more flavors than Ubuntu now... I think.